docs/markdown/Dependencies.md

```diff
@@ -282,6 +282,15 @@ in the build system DSL or with a script, likely calling
 putting these in meson upstream the barrier of using them is lowered, as
 projects using meson don't have to re-implement the logic.
 
+## Builtin
+
+Some dependencies provide no valid methods for discovery on some systems,
+because they are provided internally by the language. One example of this is
+intl, which is built into GNU or musl libc but otherwise comes as a `system`
+dependency.
+
+In these cases meson provides convenience wrappers for the `system` dependency,
+but first checks if the functionality is usable by default.
```

Comment:
NIce.

This, and the previous section 'system' are clearly misplaced here (in a list of dependency names which get special handling), and should be up in the list of detection methods.

Reply:
And config-tool is half one and half the other :/